packages/react-components/src/components/AttachmentCard.tsx

```diff
@@ -149,16 +150,64 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
                 }).iconName
               }
             />
-          </Stack>
-          <Stack className={attachmentNameContainerClassName}>
-            <Text className={attachmentNameTextClassName}>{attachmentName}</Text>
-          </Stack>
-          <Stack verticalAlign="center" className={actionIconClassName}>
-            {actionIcon && actionIcon}
-          </Stack>
-        </Stack>
-        {showProgressIndicator && <ProgressIndicator percentComplete={progress} styles={progressIndicatorStyles} />}
-      </Stack>
+          }
+          header={
+            <div className={fileNameContainerClassName}>
+              <Text title={attachmentName}>{attachmentName}</Text>
+            </div>
+          }
+          action={getMenuItems(menuActions, attachment, onDownloadErrorMessage)}
+        />
+      </Card>
+      {isUploadComplete ? (
+        <CardFooter>
+          <ProgressBar thickness="medium" value={progress} shape="rounded" />
+        </CardFooter>
+      ) : (
+        <> </>
+      )}
     </div>
   );
 };
+
+const getMenuItems = (
+  menuActions: AttachmentMenuAction[],
+  attachment: AttachmentMetadata,
+  onDownloadErrorMessage?: (errMsg: string) => void
```

Comment:
```suggestion
  handleOnClickError?: (errMsg: string) => void
```

Reply:
API to be re-worked by John, temporary placeholder to get the new FluentUi Card working

Reply:
Isn't this an internal function?